pandas/tests/extension/test_arrow.py

```diff
@@ -234,6 +235,29 @@ def test_astype_str(self, data, request):
             )
         super().test_astype_str(data)
 
+    @pytest.mark.parametrize(
+        "nullable_string_dtype",
+        [
+            "string[python]",
+            pytest.param(
+                "string[pyarrow]", marks=td.skip_if_no("pyarrow", min_version="1.0.0")
+            ),
+        ],
+    )
+    def test_astype_string(self, data, nullable_string_dtype):
```

Comment:
Could you move this test to `pandas/tests/extension/test_string.py`?

Reply:
never mind, after changing to the Series behavior this test is removed entirely